### DISCUSSION TITLE
streamFetcherRestartPeriod

### DISCUSSION THREAD

**Discussion by cjags** [ant-media/Ant-Media-Server#5511] [Q&A]
Hi

If this is non zero, what happens to streams that are already being fetched? They also restrart? Please clarify. 

Thanks
JK

**Comment by yashtandon113** [comment]
@cjags 
If you wish to restart all stream sources after a certain amount of time, you can utilize this parameter by altering its value as per your requirements. By default, `streamFetcherRestartPeriod` is set to 0, which implies that streams won't restart automatically.
